src/ingest_validation_tools/directory-schemas/mibi.yaml

```
@@ -1,3 +1,45 @@
 -
-  pattern: 'TODO'
-  description: 'Directory structure not yet specified.'
+  pattern: 'mcd/[^/]+_HuBMAP_[^/]+_slide[^/]+\.zip'
+  description: 'CSV containing labels for sections as well as whether or not they were included in the 3D model.'
+  required: False
+-
+  pattern: 'mcd/section_report\.csv'
+  description: 'Contains tissue id, acquisition id, 3D image ordering, MCD image ordering, and boolean if used for 3D model.'
+  required: False
+-
+  pattern: 'mcd/channelnames_report\.csv'
+  description: 'Contains antibodies names used and whether they were detected sufficiently or not.'
+  required: False
+-
+  pattern: '3D_image_stack\.ome\.tiff'
+  description: 'OME.tiff file comprising all slices and channels.'
+  required: True
+-
+  pattern: 'SingleCellData/cells\.csv'
+  description: 'Contains one csv file per tissue with marker intensities (untransformed, range normalized to 99th percentile), phenograph cluster label and cell type label per single cell.'
+  required: True
+-
+  pattern: 'SingleCellData/cellsinfo\.txt'
+  description: 'Text file containing formatting information about cellsorgan.csv. File is optional.'
```

```suggestion
  description: 'Text file containing formatting information about cellsorgan.csv.'
```
Preference to only use the `required` field: When information is in two places it can get out of sync. Could generate `File is optional.` as part of the description, if you think that improves usability.